Missing text in dark mode bug fixed

Carlos-Gaxiola opened the pull request:
Closes #238 

Fixed the bug that didn't allow the text to change when the dark mode was activated.

Before this fix, the text stayed in the same color when the mode was switched into dark mode and it gave the impression that it had disappeared.

The underline element that marks that it is a link, had a conflict that override the class that allows to change when the dark mode is activated, so, I created a class to the underline element that applies this change.

![Screen Shot 2021-12-16 at 5 31 42 p m](https://user-images.githubusercontent.com/63618776/146469014-4d9e065c-dd47-406c-93a2-a78d10dd6ff4.png)
On the component "StopSection" where this text is placed we have this underline element that has the conflict with the switch color class.

![Screen Shot 2021-12-16 at 5 34 14 p m](https://user-images.githubusercontent.com/63618776/146469221-96629087-6651-41d3-87e0-33147360f916.png)
This is the class that I added to apply the change when the theme is switched

Carlos-Gaxiola committed:
Missing text in dark mode bug fixed

pociust commented:
@Carlos-Gaxiola sorry for the delay in my response/review, had quite the weekend/ending of the week. I looked at the fix and it looks good, ONLY suggestion I would make is adding a transition to it so it does not just pop in. If you look at the variables.sass I believe there is a global variable ($theme-fade-speed) you can prob add to it

Carlos-Gaxiola commented:
Nice, thank you for your time to review the PR and answering, I will be working on that suggestion :)

Carlos-Gaxiola committed:
Color transition

Carlos-Gaxiola commented:
> @Carlos-Gaxiola sorry for the delay in my response/review, had quite the weekend/ending of the week. I looked at the fix and it looks good, ONLY suggestion I would make is adding a transition to it so it does not just pop in. If you look at the variables.sass I believe there is a global variable ($theme-fade-speed) you can prob add to it

Hello, I did a commit where I added the transition.

pociust commented:
@Carlos-Gaxiola Awesome! Sorry to nitpick here: correct fix, wrong placement. Unless you are trying to change all files (global) lets try and keep the scope local, you can prob add your css transition to the a.text-underline class you made in StopSection.vue to 1. save retyping code and 2. keep things scoped appropriately. Once you made that change/remove the global rule we should be good to go!

Carlos-Gaxiola committed:
Variable scope fixed

Carlos-Gaxiola commented:
> @Carlos-Gaxiola Awesome! Sorry to nitpick here: correct fix, wrong placement. Unless you are trying to change all files (global) lets try and keep the scope local, you can prob add your css transition to the a.text-underline class you made in StopSection.vue to 1. save retyping code and 2. keep things scoped appropriately. Once you made that change/remove the global rule we should be good to go!

Hello, thank you for your comments, I changed the placement and removed the global rule :)

pociust reviewed (approved):
Looks great, good job

Carlos-Gaxiola commented:
> Looks great, good job

Nice, thank you!
